app/phonons/initphonons.F90

```diff
@@ -1126,7 +1137,8 @@ subroutine readCp2kHessian(child)
 
   end subroutine readCp2kHessian
 
-  ! Subroutine removing entries in the Dynamical Matrix.
+
+  !> Subroutine removing entries in the Dynamical Matrix.
   ! Not used because identified as a wrong way
```

Comment:
Shouldn't it be `!! Not...`? If not we might also ask whether lines 1486 and 1638 might also only require one `!`.